proxy-apps/Kripke/src/Kripke/Sweep_Solver.cpp

```diff
@@ -84,13 +84,17 @@ int SweepSolver (Grid_Data *grid_data, bool block_jacobi)
   // Marks when to print a timer in simulation
   if(!mpi_rank) {
     SCOREP_USER_REGION_BY_NAME_BEGIN("TRACER_WallTime_kripke", SCOREP_USER_REGION_TYPE_COMMON);
-    SCOREP_USER_REGION_BY_NAME_BEGIN("TRACER_WallTime_kripke_region", SCOREP_USER_REGION_TYPE_COMMON);
   }
 #endif
 
   // Loop over iterations
   double part_last = 0.0;
   for(int iter = 0;iter < grid_data->niter;++ iter){
+#if WRITE_OTF2_TRACE
+    if(!mpi_rank) {
+      SCOREP_USER_REGION_BY_NAME_BEGIN("TRACER_WallTime_kripke_region", SCOREP_USER_REGION_TYPE_COMMON);
```

Comment:
This is incorrect: end is added only every few iters while begin is added every iter.